packages/query-store/package.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
what's the purpose of this package given that it's empty right now?
Is the idea to move the `QueryStore` code in evidence here down the road?